keyboards/keycapsss/o4l_5x12/o4l_5x12.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+  #define LAYOUT_o4l5x12_grid( \
```

Comment:
```suggestion
  #define LAYOUT_ortho_5x12_grid( \
```

Reply:
I think just `LAYOUT_ortho_5x12` is sufficient (`grid` is redundant to `ortho_5x12` IMO).